app/services/process_interaction_service.rb

```diff
@@ -123,7 +123,9 @@ def add_post!(body, account)
   end
 
   def status(xml)
-    Status.find(TagManager.instance.unique_tag_to_local_id(activity_id(xml), 'Status'))
+    uri = activity_id(xml)
+    return nil unless TagManager.instance.local_id?(uri)
+    Status.find(TagManager.instance.unique_tag_to_local_id(uri, 'Status'))
```

Comment:
Is there a reason that you would call unique_tag_to_local_id *without* a local URI? Then shouldn't we just guard that method appropriately to avoid further footguns?